addons/event/views/event_views.xml

```diff
@@ -197,7 +197,7 @@
             <field name="name">event.event.form</field>
             <field name="model">event.event</field>
             <field name="arch" type="xml">
-                <form string="Events">
+                <form string="Events" class="o_event_form_view">
```

Comment:
Won't this break the tour in stable if the JS file is refreshed but the module is not updated with a "-u" ?
Not sure it's ok to do in 14.0.